style.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins');
+
+@media screen and (min-width: 480px) {
+  body {
+    background-color: lightgreen;
+  }
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  overflow: hidden;
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.flexbox {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px;
+  background-color: #fff;
+}
+
+.logo {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  padding: 10px 12px;
+  text-decoration: none;
+}
+
+.fa-bars {
+  color: #6070ff;
+  width: 18px;
+  height: 12px;
+}
+
+.intro {
+  background-image: url('img/hbackground.png');
+  border-radius: 0 0 0 50px;
+  background-color: #fff;
+  padding: 20px;
+}
+
+.description {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.action-text {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  text-decoration: none;
+}
+
+.social-media li {
+  display: inline;
+}
+
+.social-media {
+  list-style-type: none;
+  padding-left: 0;
+}
```

Comment:
- [ ] Please make sure you give a gap between social media icons.

your design👇
![gap](https://user-images.githubusercontent.com/66992901/191592456-5d664d58-19ef-47fe-be12-f22f2f81949e.PNG)

Figma design 👇
![figmsocial](https://user-images.githubusercontent.com/66992901/191592535-384b9eab-bc4b-40ba-abd0-3f7058fabc18.PNG)